ui/src/features/Editor/components/OverlayUI/index.tsx

```diff
@@ -88,12 +92,15 @@ const OverlayUI: React.FC<OverlayUIProps> = ({
           </div>
         </div>
         <div id="right-top" className="absolute top-4 right-4" />
-        <div className="pointer-events-none absolute inset-y-2 bottom-4 left-4 flex items-end">
-          <DebugLogs />
+        <div
+          className={`pointer-events-none absolute inset-y-2 ${fullscreenDebug ? "bottom-0 left-0" : "bottom-4 left-4 "}   z-30 flex items-end`}>
```

Comment:
If possible, I'd prefer keeping the absolute positioning static here, and then in the DebugPanel handle all DebugPanel specific positioning when in different states. Keeps the OverlayUI more simple while still allowing us to do unique things with the component itself.
Not super important though, so if you move the positioning to DebugPanel, negate the space in bottom and left, and it doesn't really work as expected you can leave this.